server/sner/plugin/auror_hostnames/agent.py

```diff
@@ -0,0 +1,71 @@
+# This file is part of sner4 project governed by MIT license, see the LICENSE.txt file.
+"""
+sner agent auror_hostnames module
+"""
+import json
+import os
+import shutil
+from pathlib import Path
+from schema import Schema
+
+from sner.agent.modules import ModuleBase
+from sner.plugin.auror_hostnames.core import get_zone_file_paths, get_records, process_cnames, process_ptrs, clone_dns_repo, get_repos
+
+
+class AgentModule(ModuleBase):
+    """
+    auror_hostnames module implementation
+
+    ## target specification
+    target = simple-target
+    """
+
+    CONFIG_SCHEMA = Schema({"module": "auror_hostnames", "args": str})
+
+    def run(self, assignment):
+        """simply write assignment and return"""
+
+        super().run(assignment)
+
```

Comment:
also, in this particular case, either put also all logic into core module. now it's kind of split between files